### PR TITLE
feat: add xrpl ticket support for ripple handler

### PR DESCRIPTION
**Pull request by Polybius93**
This PR adds XRPL Ticket functionality to RippleHandler.

### CODE REVIEW

File: src/network-handlers/ripple-handler.ts [hunk 1]
```diff
@@ -150,12 +155,102 @@ export class RippleHandler {
...
+
+        const meta = submitCreateTicketTransactionResponse.result.meta;
+
+        if (!meta) {
+          throw new RippleError('Transaction Metadata not found');
+        }
+
+        if (typeof meta === 'string') {
+          throw new RippleError(`Could not read Transaction Result of: ${meta}`);
+        }
```
Comment (sosaucily) [inline src/network-handlers/ripple-handler.ts:179]: I don't think we need to compare and check all this stuff. Meta is always present, no? do you expect meta to not exist or to not be a string?

File: src/network-handlers/ripple-handler.ts [hunk 2]
```diff
@@ -150,12 +155,102 @@ export class RippleHandler {
...
+            Math.ceil(preparedCreateTicketTransaction.LastLedgerSequence! / 10000 + 1) * 10000;
+        }
+
+        console.log('preparedCreateTicketTransaction ', preparedCreateTicketTransaction);
+
+        const createTicketTransactionSignature = this.wallet.sign(
+          preparedCreateTicketTransaction,
+          true
+        ).tx_blob;
+        console.log('createTicketTransactionSignature: ', createTicketTransactionSignature);
```
Comment (sosaucily) [inline src/network-handlers/ripple-handler.ts:228]: remove later

File: src/network-handlers/ripple-handler.ts (hunk 1 above)
Comment (Polybius93) [reply src/network-handlers/ripple-handler.ts:179]:
```
/** Transaction metadata, which describes the results of the transaction.
   *  Can be undefined if a transaction has not been validated yet. */
  meta?: TransactionMetadata<T> | string
  ```
  
I need to verify that it is defined and is an object before treating it as one.

File: src/network-handlers/ripple-handler.ts (hunk 2 above)
Comment (Polybius93) [reply src/network-handlers/ripple-handler.ts]: Yes, I removed it!

**Review by sosaucily — APPROVED** [review]

**Pull request merged**